### ISSUE TITLE
Sending raw data is impossible.

### ISSUE THREAD

**Issue by reflog** [pleerock/routing-controllers#57]
Hi!
Thanks for an awesome project! It make writing services in TS a real pleasure.
One bug I came across - even for actions that are marked with a different @ContentType and not inside @JsonController - the result cannot be binary, for example, I cannot send image file.
The reason for it is the following code in ExpressDriver:
```
 } else {
                    response.send(String(result));
                }
```

Basically, even if a Buffer is returned - it's converted to string. It's not a correct behavior. At least some sort of escape hatch is needed. 

What would you recommend?

**Comment by awaragi** [comment]
I had to return images. Easiest thing is use @Res() to inject response object and write directly through it

**Comment by reflog** [comment]
@awaragi - do you have a code snippet to demonstrate, maybe?

**Comment by awaragi** [comment]
@Get(":I'd") get(@Param("id") id: string, @Res() response: any) {
        return this.documentDAO.get(id).then((data) => {
            if (data) {
                response.append("Content-Type", data.mimeType);
                response.end(data.file.buffer, 'UTF-8');
            } else {
                response.status(404);
                // 404 handler will take care of message content
            }
        }).catch((error) => {
            response.status(500);
            return Status.failure(error.message);
        })
    }

**Comment by pleerock** [comment]
this should be fixed in 0.7.0
